spring-integration-file/src/main/java/org/springframework/integration/file/config/AbstractRemoteFileInboundChannelAdapterParser.java

```diff
@@ -49,7 +51,8 @@ protected final BeanMetadataElement parseSource(Element element, ParserContext p
 
 		// configure the InboundFileSynchronizer properties
 		BeanDefinition expressionDef = IntegrationNamespaceUtils.createExpressionDefinitionFromValueOrExpression(
-				"remote-directory", "remote-directory-expression", parserContext, element, false);
+				"remote-directory", "remote-directory-expression", parserContext,
+				element, false);
```

Comment:
The line exceeds 120 symbols.

I feel like there might be a lot of code which we won't touch in the future at all.
But its problem that it might be inconsistent with our code style.
So, if we request 120 line length we should prove that with all our code.
Otherwise we may meet some nasty commentators that we don't do that in our own code. 😄 

I know that you don't like such a code style reformatting for nothing, but sometimes doing that via IDEA tool I meet some interesting violations, like extra whitespaces in the code line, missed blank lines around class members etc...